src/main/resources/avro/rnaquantification.avdl

```diff
@@ -1,140 +1,95 @@
-@namespace("org.ga4gh.models")
-/**
-This protocol defines feature expression counts on GA4GH reads. 
-*/
-protocol RnaQuantifications {
-
-import idl "common.avdl";
-import idl "reads.avdl";
-
-/**
-Units for expression level.
-FPKM - number of Fragments Per Kilobase of feature length per Million reads
-FPKM is calculated by dividing the fragment count per feature by the total number of reads in millions (FPM - Fragments Per Million).  FPM is then divided by feature length in kilobases to obtain FPKM.
-
-TPM - Transcripts per kilobase Per Million reads
-TPM is calculated by first dividing the fragment/read count by feature length in kilobases (RPK - Reads Per Kilobase).  The count of all RPKs in the sample are then divided by a million to generate a 'per million' scaling value.  For each feature RPK divided by the 'per million' scaling factor generated TPM.
-*/
+syntax = "proto3";
+
+package ga4gh;
+
+import "ga4gh/common.proto";
+import "ga4gh/metadata.proto";
+import "google/protobuf/struct.proto";
+
+// Units for expression level.
+// FPKM - number of Fragments Per Kilobase of feature length per Million reads
+// FPKM is calculated by dividing the fragment count per feature by the total
+// number of reads in millions (FPM - Fragments Per Million).  FPM is then
+// divided by feature length in kilobases to obtain FPKM.
+
+// TPM - Transcripts per kilobase Per Million reads
+// TPM is calculated by first dividing the fragment/read count by feature length
+// in kilobases (RPK - Reads Per Kilobase).  The count of all RPKs in the sample
+// are then divided by a million to generate a 'per million' scaling value.  For
+// each feature RPK divided by the 'per million' scaling factor generated TPM.
+
 enum ExpressionUnits {
-  FPKM,
-  TPM
+  FPKM = 0;
```

Comment:
Note: FPKM will be the default value if left uninitialized in an ExpressionLevel message. If this is not acceptable, then use something like EXPRESSION_UNIT_UNSPECIFIED as the default value. I use this pattern in a few places in my proto branch PR: https://github.com/ga4gh/schemas/pull/547

I don't have enough context to make a judgement here, but I'd guess you would want to use the unspecified enum value approach.